src/plugins/intel_gna/src/backend/gna_limitations.cpp

```diff
@@ -753,6 +753,9 @@ bool Limitations::is_transpose_supported(const ov::Shape& shape) {
             max_input_dim <= kTransposeMaxSize) {
             return true;
         }
+    } else if (squeezed_shape.size() == 1) {
```

Comment:
I suggest to remove this check from here and use is_one_dim_shape check where necessary instead. This seems to be a better solution for future use of is_transpose_supported function.

Reply:
I thought about that, it would require to add this check in many places. And function should return true for such a transpose anyway (even if transpose does nothing)